### ISSUE TITLE
"Focus Console Output" shortcut key should be replaced with easier one

### ISSUE THREAD

**Issue by jooyoungseo** [rstudio/rstudio#6850]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.3.957
    OS Version      : Windows
    R Version       : any

### Steps to reproduce the problem

Using only one hand, try to press the shortcut key (Alt+Shift+2) to "Focus Output Console."

### Describe the problem in detail

I have found the shortcut key for "Focus Output Console" (Alt+Shift+2) is very hard to press with one hand. It would've not been a big deal if the key was for changing settings (like Alt+Shift+[ "Toggle Tab Key Always Moves Focus"). However, the "Focus Console Output" key should be regarded as a *navigation* key, and this will be used as frequently by screen-reader users as "Move cursor to Console" (Ctrl+2) and "Move cursor to Source Editor" (Ctrl+1).

### Describe the behavior you expected

Would there be any way for us to consider assigning much easier shortcut key like ``'Alt + `'``? ``'Ctrl + `'``, for example, is used in Emacs and VSCode to switch between text editing mode and terminal output.

- [ ] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by gtritchie** [comment]
Too late for initial 1.3 release, but would like to get into first 1.3-patch release. Probably support the current shortcut plus this new one (at least on Windows/Linux).

**Comment by jmcphers** [comment]
We could unbind Show Request Log for this as I'm sure any use of <kbd>Ctrl</kbd> <kbd>`</kbd> outside the RStudio team is likely unintentional!

**Comment by gtritchie** [comment]
I did some research and there are some common keyboard layouts that don't even have a physical backtick key (for example, an Italian keyboard), so not a good choice for an important shortcut.

**Comment by gtritchie** [comment]
I should look into what VSCode does with this. They might localize their shortcuts and have different ones based on input locale or something like that.

**Comment by gtritchie** [comment]
Fixed in 1.3-patch via #6857

**Comment by R-Hannibal** [comment]
@jooyoungseo - Looking for help with another verification (I repeat: thanks so much for the assistance :)

Should be live in the [latest preview](https://rstudio.com/products/rstudio/download/preview/).

**Comment by jooyoungseo** [comment]
@R-Hannibal, also verified.
